### PR TITLE
CO-3128 payment_return_sepa

### PR DESCRIPTION
**Pull request by quentingigon**
This migrates the modules :

- `l10n_ch_payment_return_sepa`
  - depends on `account_payment_return_import_iso20022`
  - remove dependency on `fds_postfinance` and move this to module `l10n_ch_fds_import_payment_return` 
- `l10n_ch_fds_import_payment_return`
  - depends on `fds_postfinance` and `l10n_ch_payment_return_sepa`

Depends on #43

### CODE REVIEW

**Comment by quentingigon** [comment]
Error:
**odoo.exceptions.UserError: ("You try to install module 'l10n_ch_payment_return_sepa' that depends on module 'account_payment_return_import_sepa_pain'.\nBut the latter module is not available in your system.", '')**

This PR needs this one to pass -> [HERE](https://github.com/CompassionCH/account-payment/pull/2)

**Comment by ecino** [comment]
I created the branch for payment return sepa. Can you please rebase and keep only relevant commits? Thanks.

**Comment by quentingigon** [comment]
Need this PR first -> [HERE](https://github.com/CompassionCH/account-payment/pull/2)

**Comment by ecino** [comment]
The parent module was renamed to `account_payment_return_import_iso20022`! (see my comment in your Pull request on account-payment repository). Please adapt to these changes.

**Comment by quentingigon** [comment]
depends on module _l10n_ch_fds_postfinance_ that is in this PR -> [HERE](https://github.com/CompassionCH/l10n-switzerland/pull/43)

File: l10n_ch_payment_return_sepa/demo/demo.yml [hunk 1]
```diff
@@ -0,0 +1,505 @@
+#-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+# Create in invoices
+#-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+-
+  !record {model: account.invoice, id: ct_invoice_1, view: False}:
```
Comment (ecino) [inline l10n_ch_payment_return_sepa/demo/demo.yml]: Yaml is no more supported. Remove the file and make sure nothing is missing for the tests. If needed, convert this to xml

**Review by ecino — CHANGES_REQUESTED** [review]

File: l10n_ch_payment_return_sepa/models/pain002_parser.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
+import re
+
+from lxml import etree
+
+from odoo import models
+from odoo.addons.account_payment_return_import_sepa_pain.wizard.pain_parser \
```
Comment (ecino) [inline l10n_ch_payment_return_sepa/models/pain002_parser.py]:
```suggestion
from odoo.addons.account_payment_return_import_iso20022.wizard.pain_parser \
```

**Review by ecino — CHANGES_REQUESTED** [review]

**Comment by quentingigon** [comment]
I will make this PR pass when **l10n_ch_fds_postfinance** will be merged with branch 12.0 or 12.0-everything-migrated

File: l10n_ch_payment_return_sepa/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    'name': 'Parse payment return',
+    'version': '12.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'Compassion CH, Odoo Community Association (OCA)',
+    'website': 'https://www.compassion.ch',
+    'category': 'Banking addons',
+    'depends': [
+        'account_payment_order',  # OCA/bank-payment
+        'account_payment_return_import_iso20022',  # OCA/account-payment
+        'l10n_ch_fds_postfinance',
```
Comment (ecino) [inline l10n_ch_payment_return_sepa/__manifest__.py]: You didn't move the dependency as I asked

File: l10n_ch_payment_return_sepa/models/__init__.py [hunk 4]
```diff
@@ -0,0 +1,4 @@
+from . import pain002_parser
+from . import fds_postfinance_file
```
Comment (ecino) [inline l10n_ch_payment_return_sepa/models/__init__.py]: This should be moved into fds_import_payment_return module

File: l10n_ch_payment_return_sepa/models/__init__.py [hunk 5]
```diff
@@ -0,0 +1,4 @@
...
+from . import errors
+from . import fds_postfinance_directory
```
Comment (ecino) [inline l10n_ch_payment_return_sepa/models/__init__.py]: This should be moved out of the module

File: l10n_ch_payment_return_sepa/models/pain002_parser.py (hunk 2 above)
Comment (ecino) [reply l10n_ch_payment_return_sepa/models/pain002_parser.py]: You reverted the change and went back to the wrong version!

**Review by ecino — CHANGES_REQUESTED** [review]

File: l10n_ch_payment_return_sepa/__manifest__.py (hunk 3 above)
Comment (quentingigon) [reply l10n_ch_payment_return_sepa/__manifest__.py]: Sorry, I think I did it on an other branch

**Comment by ecino** [comment]
Manually merged.

**Pull request closed without merging**